pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -138,223 +138,223 @@
 <!ENTITY pbContentTextContextMenu-thingsNotesMac.accesskey   "T">
-<!ENTITY pbContentTextContextMenu-thingsTitleMac.label       "Things: New Reminder with Selection as Title">
+<!ENTITY pbContentTextContextMenu-thingsTitleMac.label       "Things: novo lembrete com seleção como Título">
 <!ENTITY pbContentTextContextMenu-thingsTitleMac.accesskey   "i">
-<!ENTITY pbContentTextContextMenu-omniFocusMac.label       "OmniFocus: Send to Inbox">
+<!ENTITY pbContentTextContextMenu-omniFocusMac.label       "OmniFocus: enviar à Caixa de Entrada">
 <!ENTITY pbContentTextContextMenu-omniFocusMac.accesskey   "i">
 
 <!-- topic context menu -->
-<!ENTITY pbTopicContextMenu-edit.label         "Rename">
+<!ENTITY pbTopicContextMenu-edit.label         "Renomear">
 <!ENTITY pbTopicContextMenu-edit.accesskey     "e">
-<!ENTITY pbTopicContextMenu-favorite.label     "Favorite Topic">
+<!ENTITY pbTopicContextMenu-favorite.label     "Tópico favorito">
 <!ENTITY pbTopicContextMenu-favorite.accesskey "F">
-<!ENTITY pbTopicContextMenu-remove.label       "Remove">
+<!ENTITY pbTopicContextMenu-remove.label       "Remover">
 <!ENTITY pbTopicContextMenu-remove.accesskey   "R">
-<!ENTITY pbTopicContextMenu-editAll.label      "Edit Topics...">
+<!ENTITY pbTopicContextMenu-editAll.label      "Editar tópicos...">
 <!ENTITY pbTopicContextMenu-editAll.accesskey  "t">
 
 <!-- account tree context menu -->
-<!ENTITY pbAccountTreeContextMenu-newGroup.label "New Account Group...">
+<!ENTITY pbAccountTreeContextMenu-newGroup.label "Novo grupo de contas...">
 <!ENTITY pbAccountTreeContextMenu-newGroup.accesskey "G">
-<!ENTITY pbAccountTreeContextMenu-settings.label "Settings...">
+<!ENTITY pbAccountTreeContextMenu-settings.label "Configurações...">
 <!ENTITY pbAccountTreeContextMenu-settings.accesskey "i">
 
 <!-- folder pane -->
-<!ENTITY accountTreeMac.label    "ACCOUNTS">
-<!ENTITY folderTreeMac.label     "FOLDERS">
+<!ENTITY accountTreeMac.label    "CONTAS">
+<!ENTITY folderTreeMac.label     "PASTAS">
 
 <!-- thread pane -->
-<!ENTITY threadPaneNoMatchesFound.desc "No messages matched your search.">
-<!ENTITY threadPaneNoMatchesSuggestionAllFolders.desc "Try looking in All Folders.">
-<!ENTITY threadPaneNoMatchesSuggestionAllFocus.desc "Try setting your Focus to All.">
-<!ENTITY threadPaneNoMatchesSuggestionAllMessages.desc "Try setting your view to Messages.">
-<!ENTITY threadPaneAddTextToSearch.desc "Start typing in the search bar to begin your search.">
+<!ENTITY threadPaneNoMatchesFound.desc "Nenhuma mensagem corresponde aos critérios da pesquisa.">
+<!ENTITY threadPaneNoMatchesSuggestionAllFolders.desc "Tente verificar em Todas as Pastas.">
+<!ENTITY threadPaneNoMatchesSuggestionAllFocus.desc "Tente definir seu foco para Todos.">
+<!ENTITY threadPaneNoMatchesSuggestionAllMessages.desc "Tente definir sua exibição para Mensagens.">
+<!ENTITY threadPaneAddTextToSearch.desc "Comece a digitar na barra de pesquisa para iniciar sua pesquisa.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY threadPaneAddTextToSearch.desc "Comece a digitar na barra de pesquisa para iniciar a busca.">